internal/subscriber/retry/retrier_test.go

```diff
@@ -87,3 +104,69 @@ func Test_Interval_Calculator(t *testing.T) {
 		assert.Equal(t, test.want, actualIntervals)
 	}
 }
+
+func TestRetrier_Start(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	ctx := context.Background()
+	ctxWt, cancel := context.WithTimeout(ctx, time.Second*1)
+	defer cancel()
```

Comment:
add defer block as the first func statement followed by a new line. Easy to spot

Reply:
Addressed